docs/book/compatiblity_matrix.md

```diff
@@ -1,16 +1,20 @@
 # Compatibility Matrix for vSphere CSI Driver
 
-- vSphere CSI driver is released with specific version of CSI sidecar containers which has the minimum kubernetes version requirement.
+- vSphere CSI driver is released with specific version of CSI sidecar containers which has the minimum and maximum kubernetes version requirement.
 
-| vSphere CSI Driver | Minimum Kubernetes Version |
-|--------------------|----------------------------|
-| v1.0.2             | v1.14.0                    |
-| v1.0.3             | v1.14.0                    |
-| v2.0.0             | v1.16.0                    |
-| v2.0.1             | v1.17.0                    |
-| v2.1.0             | v1.17.0                    |
+Note:
 
-- vSphere CSI driver is compatible with vSphere 67u3, vSphere 7.0 and 7.0u1.
+- vmware supports the deprecated release of the driver until End Of Life date.
+
+| vSphere CSI Driver | Minimum Kubernetes Version | Maximum Kubernetes Version            | Deprecated | End of Life  |
+|--------------------|----------------------------|---------------------------------------| -----------|--------------|
+| [v2.1.0](./releases/v2.1.0.md)             | v1.17.0                    | v1.19.0       | No         | -            |
```

Comment:
Can we mention the Kubernetes release like v1.17, v1.18, etc? If a customer is running v1.19.5 of k8s, then this table says that it is not supported even though we have tested with a v1.19.x patch release.

Reply:
done.